docs/oid-info.md

```diff
@@ -189,27 +189,27 @@ that Sigstore operates.
+| iss                | iss     | iss       | iss        | iss        | Issuer                       | This already exists. For example: https://token.actions.githubusercontent.com                                                                                                          |
+| exp                | exp     | exp       | exp        | exp        | N/A                          | Only used to validate the JWT.                                                                                                                                                         |
+| nbf                | nbf     | nbf       | nbf        | nbf        | N/A                          | Only used to validate the JWT. Optional, as per the OIDC spec                                                                                                                         |
+| iat                | iat     | iat       | iat        | iat        | N/A                          | Only used to validate the JWT.                                                                                                                                                         |
+| server_url + job_workflow_ref   | ci_config_ref_uri ([WIP][gitlab-wip-cliams]) | ??       | ??        | platform_url + /build/ +  workflow_id        | Build Signer URI             | Reference to specific build instructions that are responsible for signing. Can be the same as Build Config URI. For example a reusable workflow in GitHub Actions or a Circle CI Orbs. |
+| job_workflow_sha   | ci_config_sha ([WIP][gitlab-wip-cliams]) | ??       | ?? | N/A        | Build Signer Digest          | An immutable reference to the specific version of the build instructions that is responsible for signing. Should include the digest type followed by the digest, e.g. `sha1:abc123`.   |
+| runner_environment | runner_environment | ??       | ??        | runner_environment        | Runner Environment           | For platforms to specify whether the build took place in platform-hosted cloud infrastructure or customer-hosted infrastructure. For example: `platform-hosted` and `self-hosted`.     |
+| server_url + repository         | server_url + project_path | ??       | ??        | scm_repo_url        | Source Repository URI                   | Should include a fully qualified repository URL.                                                                                                                                       |
+| sha                | sha | ??       | build_commit | N/A | Source Repository Digest                | An immutable reference to a specific version of the source code. Should include the digest type followed by the digest, e.g. `sha1:abc123`.                                            |
```

Comment:
Out of scope for this PR, but an ask for Codefresh: it would be great if this could start to be captured in OIDC claims so that users can know what version of the pipeline config was used for the build.

Reply:
Thank you for your detailed review @wlynch! We will for sure discuss it internally and try to come up with a relevant claim.

Reply:
@steiza We will for sure take it into account. I'll work with the relevant dev team to have a relevant claim added in the near future, and will add it as soon as it is available.